examples/qml_features/rust/src/multiple_qobjects.rs

```diff
@@ -0,0 +1,61 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+#[cxx_qt::bridge(cxx_file_stem = "multiple_qobjects")]
+mod ffi {
+    #[cxx_qt::qobject]
+    pub struct FirstObject {
+        #[qproperty]
+        counter: i32,
+    }
+
+    impl Default for FirstObject {
+        fn default() -> Self {
+            Self { counter: 10 }
+        }
+    }
+
+    #[cxx_qt::qsignals(FirstObject)]
+    pub enum FirstSignals {
+        DataChanged,
+    }
+
+    impl qobject::FirstObject {
+        #[qinvokable]
+        pub fn increment(mut self: Pin<&mut Self>) {
+            let new_value = self.as_ref().counter() + 1;
+            self.as_mut().set_counter(new_value);
+
+            self.emit(FirstSignals::DataChanged);
+        }
+    }
+
+    #[cxx_qt::qobject]
+    pub struct SecondObject {
```

Comment:
I think in this example the SecondObject should at least have slightly different behavior from the FirstObject.
Otherwise the example makes it seem like there are just two instances of the same class.

For example, have the page show two "Click FirstObject" and "Click SecondObject" buttons.
If you click the FirstObject, the button color cycles.
If you click the SecondObject, it moves the button to the right.
Or something silly like that.

Reply:
Your fix should make this obvious enough :)

Reply:
Changed so they have a unique property and behaviour. But note that i chose having some of the same propertties/signals/invokables to test that our namespacing works correctly and doesn't collide.